tools/godotcpp.py

```diff
@@ -152,6 +152,11 @@ def scons_generate_bindings(target, source, env):
 
     api = generate_trimmed_api(str(source[0]), profile_filepath)
 
+    if "precision" in api["header"] and env["precision"] != api["header"]["precision"]:
+        raise UserError(
+            f"Cannot do a precision={env['precision']} build using '{str(source[0])}' which was generated by a Godot built with precision={api['header']['precision']}"
```

Comment:
```suggestion
            f"Cannot do a precision={env['precision']} build using '{source[0]}' which was generated by a Godot built with precision={api['header']['precision']}"
```

Explicit call of `str()` is not needed since Python automatically calls it on any object passed in a fstring


example:
```python
>>> class K:
...   def __str__(self):
...     return 'foo'
... 
>>> f"k is {K()}"
'k is foo'
```

Reply:
Thanks! I ended up moving where this message is generated, so I no longer need to print out the file object, but that's good to know for the future :-)